modules/clickhouse/mounts/zk_config.xml.tpl

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<clickhouse>
+    <zookeeper>
+        <node index="1">
+            <host>{{.Host}}</host>
+            <port>{{.Port}}</port>
+        </node>
+    </zookeeper>
+
+    <remote_servers>
+        <default>
+            <shard>
+                <replica>
+                    <host>localhost</host>
+                    <port>9000</port>
+                </replica>
+            </shard>
+        </default>
+    </remote_servers>
+    <macros>
+        <cluster>default</cluster>
+        <shard>shard</shard>
+        <replica>replica</replica>
+    </macros>
+
+    <distributed_ddl>
+        <path>/clickhouse/task_queue/ddl</path>
+    </distributed_ddl>
+
+    <format_schema_path>/var/lib/clickhouse/format_schemas/</format_schema_path>
```

Comment:
are those default values? Can we just keep the zookeeper section?

Reply:
all this is a part of zookeeper default configuration, without it cluster still considered as not replicated one.